tests/src/test/scala/system/basic/WskBasicPythonTests.scala

```diff
@@ -119,4 +122,37 @@ class CLIPythonTests
                     activation.logs.get.mkString("\n") should { not include ("pythonaction.py") and not include ("flask") }
             }
     }
+
+    behavior of "Python runtime"
+
+    it should "invoke a zipped Python action with virtualenv package" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "zippedPythonAction"
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) =>
+                    action.create(name, zippedPythonAction, kind = Some("python:2"))
+            }
+
+            withActivation(wsk.activation, wsk.action.invoke(name)) {
+                activation =>
+                    val response = activation.response
+                    response.result.get.fields.get("error") shouldBe empty
+                    response.result.get.fields.get("Networkinfo: ") shouldBe defined
+            }
+    }
+
+    it should "Ensure that zipped Python actions cannot be created without a kind specified" in withAssetCleaner(wskprops) {
```

Comment:
@dubeejw this is a test we should have in basic usage tests - is there one already?
if so, i would remove this test from this suite.
if not, i would move this test to `WskBasicUsageTests`.